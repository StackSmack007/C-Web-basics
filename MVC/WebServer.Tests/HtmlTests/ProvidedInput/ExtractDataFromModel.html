<p>Az se kazvam @model.Name</p>
<p>Razpolagam sus:</p>
@for (int i = 0; i < model.Neshta.ToArray().Length; i++)
    {
    <p><span>@(i+1)</span>.@model.Neshta[i]</p>
    }