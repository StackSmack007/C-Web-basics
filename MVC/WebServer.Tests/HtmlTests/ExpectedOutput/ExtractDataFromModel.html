<p>Az se kazvam Todor</p>
<p>Razpolagam sus:</p>
<p><span>1</span>.Qbulki</p>
<p><span>2</span>.Krushi</p>
<p><span>3</span>.Morkovi</p>